src/shared/ui/ShowMessage/ShowMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import {
+  toast as ShortMessage,
+  ToastOptions as IToastOptions,
+} from 'react-toastify';
+import { Link } from 'react-router-dom';
+
+import css from './ShowMessage.module.scss';
+
+export const ShowMessage = (message: string, options: IToastOptions) => {
+  ShortMessage(
+    <span>
+      {message}
+      <Link to="/cart" className={css.messageLink}>
+        View
+      </Link>
+    </span>,
```

Comment:
this span is component so it should be separated in another file 

ShortMessage  don't return TSX, actually it returns id of popup
ShowMessage returns undefined

toast have to be just a function triggered popup the better place for it is shared/lib